assessment.sql

```diff
@@ -5,24 +5,49 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+  id SERIAL PRIMARY KEY
+  surname CHARACTER VARYING
+  given_name CHARACTER VARYING
+  gender CHARACTER VARYING
+  height INTEGER
+  weight INTEGER
+  dob DATE
+  eye_color CHARACTER VARYING
+  hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees(id,surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES('Bishop','Caren','f',63,132,'1943-09-26','Brown','Black');
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(id,surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true,FORMAT csv);
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT * FROM licensees WHERE eye_color = 'Hazel' && (hair_color = 'Brown' || 'Black');
```

Comment:
In SQL, logical operators are words, not symbols. You want to use `AND` and `OR` here, not `&&` and `||`. Also, after the `OR`, you need a complete comparison expression, like so: `... eye_color = 'Hazel' AND (hair_color = 'Brown' OR hair_color = 'Black') ...`

You did make good use of parentheses to override the default order of precedence, though, and the statement is otherwise well-formed. 3/5